starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +137,26 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist( arr, str){
+  //console.log(typeof(arr));
+  if( arr === false) {
+    return false;
+  }
+  let count = 0;
+  for(let i = 0; i < arr.length; i++) {
+    if(str === arr[i]){
+      count ++;
```

Comment:
Since the function is asking you to return true if the word is in the array, you can actually just return true here and exit the function. No need to iterate, as anything greater than 0 will return true anyway.